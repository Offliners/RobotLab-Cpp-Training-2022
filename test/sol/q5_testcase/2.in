(20*2)/(4-6*14*14/(13+9-10))-16+11-2+(2/7)*(13+12)=*14-6/11-15/7/15=-13+2=-17/14/2/17*(2+2)-5/14=+(9*17)*7*7+(16+8)/15/14%4=
