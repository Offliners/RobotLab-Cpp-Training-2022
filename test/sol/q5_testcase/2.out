-18
-356
-5
-18
